GitUI/UserControls/ToolStripEx.cs

```diff
@@ -18,6 +19,64 @@ public ToolStripEx()
             _gripButton = propGrip.GetValue(this) as ToolStripButton;
         }
 
+        protected override void OnItemAdded(ToolStripItemEventArgs e)
+        {
+            base.OnItemAdded(e);
+
+            if (e.Item is ToolStripDropDownItem item)
+            {
+                item.DropDownOpening += SplitButton_DropDownOpening;
+                item.DropDownClosed += SplitButton_DropDownClosed;
+            }
+        }
+
+        protected override void OnItemRemoved(ToolStripItemEventArgs e)
+        {
+            if (e.Item is ToolStripDropDownItem item)
+            {
+                item.DropDownOpening -= SplitButton_DropDownOpening;
+                item.DropDownClosed -= SplitButton_DropDownClosed;
+            }
+        }
+
+        [DllImport("user32.dll")]
+        private static extern int SendMessage(IntPtr hwnd, int wmsg, bool wparam, int lparam);
+
+        private const int WM_SETREDRAW = 11;
```

Comment:
All interop code resides in https://github.com/gitextensions/gitextensions/tree/master/GitUI/Interops. We already have definitions for `SendMessageW`:
https://github.com/gitextensions/gitextensions/blob/e78a4e1fffd17989b771ed3301aca73d384e0afd/GitUI/Interops/User32/SendMessageW.cs#L8

Reply:
I added `WM_SETREDRAW` with XML document taken from MSDN and slightly modified.

I also added `NativeMethods.TRUE` and `NativeMethods.FALSE`, I wonder why they are not already present, let me know if you see a better way.